routes/bears.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const router = express.Router();
+const Bear = require('../models/bear');
+const mongoose = require('mongoose');
+
+//mongoose.connect('mongodb://localhost/bear', { useNewUrlParser: true });
+mongoose
+	.connect('mongodb+srv://greenlad:8564@greenlad@cluster0-jonki.mongodb.net/test?retryWrites=true', { useNewUrlParser: true })
+	.then(() => console.log('Conneting to mongoDB'))
+	.catch((err) => console.log(err.message));
+
+//get verb for Bear
+router.get('/api/bears', async (req, res) => {
+	const bears = await Bear.find();
+
```

Comment:
Why this empty line?

Reply:
I am sorry about the empty line, would work on it